src/profile.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class MyForm extends React.Component {
```

Comment:
maybe change the class name to something more descriptive

Reply:
alright